mongodb-atlas-sql-interface.md

```diff
@@ -5,70 +5,102 @@
 
 ## Summary
 
-| Item                               | Description                  |
-|------------------------------------|------------------------------|
-| Release State                      | Preview                      |
-| Products                           | Power BI (datasets)          |
-| Authentication Types Supported     | Database (Username/Password) |
+| Item                               | Description                                  |
+|------------------------------------|----------------------------------------------|
+| Release State                      | Beta                                         |
+| Products                           | Power BI (Datasets)<br/>Power BI (Dataflows) |
+| Authentication Types Supported     | Database (Username/Password)                 |
 
 ## Prerequisites
-To use the MongoDB Atlas SQL connector, you must have an 
-[Atlas federated database](https://www.mongodb.com/docs/atlas/data-federation/) setup.
+To use the MongoDB Atlas SQL connector, you must have an [Atlas federated database](https://www.mongodb.com/docs/atlas/data-federation/) setup.
+
+### Obtaining connection information for your federated database instance
+
+1. Navigate to your federated database instance. In Atlas, select **Data Federation** from the left navigation panel.
+2. Select **Connect** to open the federated database instance connection modal.
+3. Select **Connect using the Atlas SQL Interface**.
+4. Select **Power BI Connector**.
+5. Copy your federated database name and MongoDB URI. You'll need them in a later step.
+
 > [!NOTE]
 > If some or all of your data comes from an Atlas cluster, you must use MongoDB version 5.0 or greater for that cluster to take advantage of Atlas SQL.
 
-We also recommend to install the MongoDB Atlas SQL ODBC Driver before using the MongoDB Atlas SQL connector.
-
+We also recommend that you install the MongoDB Atlas SQL ODBC Driver before using the MongoDB Atlas SQL connector.
```

Comment:
QQ - is this a "recommendation" or a "requirement"? I am pretty sure we should include this in the note with the requirement of Atlas 5.0. And should we include the the link to the download center?

Reply:
The ODBC driver is a requirement. You can install it after the connector. So maybe this is what this was about.
The MongoDB 5.0 requirement comes from our translations being only compatible with 5.0 and later I believe.